platform/feature-api/api/src/rdf-spec/dataset.ts

```diff
@@ -38,6 +44,9 @@ export class DatasetSpec<
         private readonly dataFactory: DataFactory<OutQuad>
     ) {}
 
+    /**
+     * It creates a new dataset with the given quads
+     */
```

Comment:
It _looks_ to me like this class inherits some interface - couldn't we also inherit the JSDoc somehow?

Reply:
mmh I might miss something here...but can't see any inherited method of this.

Reply:
Right, it doesn't - misread the type specification :D